### DISCUSSION TITLE
Keyboard usage

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#695] [General]
I'd like to use a keyboard to navigate through rAudio-1.

Using the keyboard, I can use the tab keys to hop between the music sources, the now playing screen, and the playlist.

I can use the up and down arrows to move through the playlist and hit Enter on a song to play it.

In the Library, I can use the arrow keys to move over to AlbumArtist and press Enter. I can use the arrow keys to move up and down the list, but I can't find a key on the keyboard that will let me go into an artist and see the list of albums. If I use the mouse to go into an artist and go into an album, I can use the arrow keys to move through the UI, but I can't select anything.

If I use the left arrow key, it will back out of the album list back to the artist.

I'd like to be able to navigate to an album and add it to my current playlist using just the keyboard.

If I can do this, then I should be able to buy a remote and configure the remote to do this.

**Comment by rern** [comment]
Fixed.

**Comment by apastuszak** [comment]
Thank you!
